Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -378,6 +378,19 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
 #endif
 }
 
+static void applyContentsFormatToLayer(CALayer *layer, ContentsFormat contentsFormat)
+{
+    if (RetainPtr formatString = contentsFormatString(contentsFormat))
+        [layer setContentsFormat:formatString.get()];
+#if ENABLE(PIXEL_FORMAT_RGBA16F)
+    if (contentsFormat == ContentsFormat::RGBA16F) {
+        ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+        [layer setWantsExtendedDynamicRangeContent:true];
+        ALLOW_DEPRECATED_DECLARATIONS_END
+    }
```

Comment:
suggested addition:
```
diff --git a/Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm b/Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm
index 6a6be5e4c328..aed53b4b988e 100644
--- a/Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm
+++ b/Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm
@@ -387,6 +387,7 @@ static void applyContentsFormatToLayer(CALayer *layer, ContentsFormat contentsFo
         ALLOW_DEPRECATED_DECLARATIONS_BEGIN
         [layer setWantsExtendedDynamicRangeContent:true];
         ALLOW_DEPRECATED_DECLARATIONS_END
+        [layer setToneMapMode:CAToneMapModeIfSupported];
     }
 #endif
 }
```

Reply:
This would enable tone mapping for painted content too, which I don't think we want to do.

Reply:
Does it do that or did I test incorrectly? CADebug -s layers still shows ‘never’ for the toneMapMode testing hdr images